Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Search/SearchWidget.cpp

```diff
@@ -169,10 +169,12 @@ namespace AzToolsFramework
                 SetTypeFilters(buildTypesFilterList());
             }
 
-            auto sourceFilter = new EntryTypeFilter();
-            sourceFilter->SetName("Source");
-            sourceFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Source);
-            m_projectSourceFilter->AddFilter(FilterConstType(sourceFilter));
+            auto productFilter = new EntryTypeFilter();
+            productFilter->SetName("Product");
+            productFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Product);
+            auto inverseProductFilter = new InverseFilter();
+            inverseProductFilter->SetFilter(FilterConstType(productFilter));
+            m_projectSourceFilter->AddFilter(FilterConstType(inverseProductFilter));
```

Comment:
This works. A more explicit alternative would be a composite of `Source` and `Folder` types, in case we add any additional entry types in the future (e.g. currently there is also a `Root` type, which I think you probably want to display but I'm not certain).

```
EntryTypeFilter* foldersFilter = new EntryTypeFilter();
                foldersFilter->SetEntryType(AssetBrowserEntry::AssetEntryType::Folder);
```

Reply:
Yes, that is the way it's generally done.

Reply:
Thanks, I will update it